Dockerfile

```diff
@@ -1,22 +1,18 @@
 # Mozilla Kinto server
-
-FROM node:lts-bullseye-slim as node-builder
-RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates curl
-COPY scripts/build-kinto-admin.sh .
-COPY /kinto/plugins/admin ./kinto/plugins/admin
-RUN bash build-kinto-admin.sh
-
 FROM python:3.10-slim-bullseye as python-builder
 RUN python -m venv /opt/venv
-RUN apt-get update && apt-get install -y --no-install-recommends build-essential libpq-dev
+RUN apt-get update && apt-get install -y --no-install-recommends build-essential libpq-dev ca-certificates curl
```

Comment:
Can we change the base container to `FROM python:3.10` instead? That way we can get rid of this entire line.

Reply:
Good call, makes building faster too.